UI/resources/js/main.js

```diff
@@ -296,31 +308,70 @@ function QuestionsInit() {
   questionsDetailsAjax(request);
 }
 
+/*======================================================
+                    // RSVP
+======================================================*/
+function Rsvp(checkbox) {
+  event.preventDefault();
+  // var checkBoxes = document.getElementsByName("rsvp");
+  // checkBoxes.forEach(item => {
+  //   if (item !== checkbox) item.checked = false;
+  // });
+}
+function rsvpForm() {
+  event.preventDefault();
+  notLoggedIn();
+  const response = event.target.value;
+  const meetupId = localStorage.getItem("meetupId");
+  const data = {
+    response
+  };
+  const url = `${baseUrl}/meetups/${meetupId}/rsvps`;
+  let request = post(url, data);
+
+  async function postRsvp(payLoad) {
+    try {
+      let response = await fetch(payLoad);
+      let data = await response.json();
+      if (!response.ok) {
```

Comment:
Since you are returning the response status in all the conditions, why do you need the if statements again?